insights/parsers/installed_product_ids.py

```diff
@@ -88,23 +88,39 @@ class InstalledProductIDs(CommandParser):
             Brand Type:
             Brand Name:
 
-    Filters have been added to the parser so that only the ``ID`` element will
-    be collected.
+    Filters have been added to the parser so that only the filtered lines will be collected.
 
     Attributes:
         ids (set): set of strings of the unique IDs collected by the command
+        data(list): list of dicts of the attributes and its value
```

Comment:
`data` is not a good name anymore to use as attributes, as itself cannot give any hit about what kind of the `data` is. Please rename the attribute more meaningful, e.g., `product_list`, etc.

Reply:
@xiangce sure, I'll rename it.